docs/source/notebooks/DataTransformers.md

```diff
@@ -43,6 +43,8 @@ In this notebook, we explore how to reconcile this functionality with the sklear
 * [5. Multidimensional inputs with MNIST dataset](#5.-Multidimensional-inputs-with-MNIST-dataset)
```

Comment:
I don't know if I'd include the psuedo-code in this notebook: that seems like it should be in the API documentation, not an example.

Reply:
The goal was to introduce the general structure/plan of what the notebook was going to introduce without getting bogged down in the details by introducing a fully working version.

I initially put the `if False:`s in there to avoid errors when executing, which was probably just lazy. I can probably remove all of the `if False:` by just making sure all of the pseudocode is valid python, or just marking the cell as non-executable (which is possible). Would this resolve the problem? I am not sure I am understanding how mospaced font plays into this.

Reply:
> introduce the general structure/plan of what the notebook was going to introduce without getting bogged down in the details by introducing a fully working version.

I think that goal can be accomplished without any psuedo-code. "This notebook will detail how to do the following: [list]."

Marking the cell as non-executable would resolve the complaint about `if False`

Reply:
I deleted most of the pseudocode and moved the rest to `advanced.rst`.

That said, for future reference, marking one of these markdown code cells as non executable is as easy as:

```
    ```python .noeval
    # code here
    ```
```